wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/model/impl/CommandImpl.java

```diff
@@ -88,6 +97,16 @@ public void setCommandLine(String commandLine) {
     this.commandLine = commandLine;
   }
 
+  public PreviewUrlImpl getPreviewUrl() {
+    return previewUrl;
+  }
+
+  public void setPreviewUrl(PreviewUrl previewUrl) {
+    if (previewUrl != null) {
```

Comment:
So it is not possible to remove a preview URL from a command?

Reply:
hmm, it is not. it should be
```
...
} else {
  this.previewUrl = null;
}
```
I'll fix that

Reply:
done